lighthouse-core/index.js

```diff
@@ -17,6 +17,8 @@
 
 'use strict';
 
+/* eslint-disable no-console */
```

Comment:
maybe just `// eslint-disable-next-line no-console` for just the one line in this file too?